Source/WebCore/platform/graphics/GraphicsContextGL.h

```diff
@@ -1681,6 +1708,38 @@ IMPLEMENT_GCGL_OWNED(Texture)
 
 #undef IMPLEMENT_GCGL_OWNED
 
+#if PLATFORM(COCOA)
```

Comment:
If you want to style this, I think this can be declared in the Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```
#if ENABLE(GPU_PROCESS) && ENABLE(WEBGL)
....
header: <WebCore/GraphicsContextGL.h>
#if PLATFORM(COCOA)
struct WebCore::ExternalImageSourceIOSurfaceHandle {
      MachSendRight handle;
};
struct WebCore::ExternalImageSourceMTLSharedTextureHandle {
     MachSendRight handle;
};
#endif

```

Reply:
Excellent.